apis/v1beta1/httproute_types.go

```diff
@@ -158,12 +158,13 @@ type HTTPRouteRule struct {
 	// Proxy or Load Balancer routing configuration generated from HTTPRoutes
 	// MUST prioritize matches based on the following criteria, continuing on
 	// ties. Across all rules specified on applicable Routes, precedence must be
-	// given to the match with the largest number of:
+	// given to the rule having:
```

Comment:
I think that `given to the match having` is more accurate than `given to the rule having` wording, as the field specifying the `HTTPRouteMatch` is a match actually, not a rule.

Reply:
Thanks @mlavacca. Yes that is indeed more appropriate.